### PR TITLE
qa/tests: added rbd_import_export workload

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
`CEPH_BRANCH=master; MACHINE_NAME=smithi; teuthology-suite -v -c $CEPH_BRANCH --ceph-repo https://github.com/ceph/ceph.git --suite-repo https://github.com/yuriw/ceph.git -k distro -n 7 -m $MACHINE_NAME -s upgrade/octopus-x -t master -e $CEPH_QA_EMAIL -p 50 --suite-branch wip-yuriw-octopus-x-master`

File: src/common/legacy_config_opts.h [hunk 1]
```diff
@@ -1424,6 +1424,7 @@ OPTION(rgw_relaxed_s3_bucket_names, OPT_BOOL) // enable relaxed bucket name rule
 OPTION(rgw_defer_to_bucket_acls, OPT_STR) // if the user has bucket perms)
 OPTION(rgw_list_buckets_max_chunk, OPT_INT) // max buckets to retrieve in a single op when listing user buckets
 OPTION(rgw_md_log_max_shards, OPT_INT) // max shards for metadata log
+OPTION(rgw_curl_buffersize, OPT_INT) // set preferred receive buffer size for curl calls
```
Comment (neha-ojha) [inline src/common/legacy_config_opts.h:1427]: @yuriw you've got extra changes in here

Comment (yuriw) [reply src/common/legacy_config_opts.h]: @neha-ojha fixed thx !

**Review by jdurgin — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]

**Comment by yuriw** [comment]
test this please

**Comment by yuriw** [comment]
test this please

**Comment by neha-ojha** [comment]
jenkins retest this please

**Pull request merged**